internal/controllers/topology/cluster/cluster_controller_test.go

```diff
@@ -1094,13 +1094,17 @@ func TestReconciler_DefaultCluster(t *testing.T) {
 		{
 			name: "Default Cluster variables with values from ClusterClass",
 			clusterClass: classBuilder.DeepCopy().
-				WithVariables(clusterv1.ClusterClassVariable{
-					Name:     "location",
-					Required: true,
-					Schema: clusterv1.VariableSchema{
-						OpenAPIV3Schema: clusterv1.JSONSchemaProps{
-							Type:    "string",
-							Default: &apiextensionsv1.JSON{Raw: []byte(`"us-east"`)},
+				WithStatusVariables(clusterv1.ClusterClassStatusVariable{
+					Name: "location",
+					Definitions: []clusterv1.ClusterClassStatusVariableDefinition{
+						{
+							Required: true,
```

Comment:
nit: Should we add `From: clusterv1.VariableDefinitionFromInline,` for these tests?

It looks like it's obviously not required, but seems more realistic which could be nice when we look at this later